programs/stake/src/stake_state.rs

```diff
@@ -592,8 +614,14 @@ impl Stake {
         // if there is no newer credits since observed, return no point
         if new_vote_state.credits() <= self.credits_observed {
             if fix_stake_deactivate {
+                if let Some(inflation_point_calc_tracer) = inflation_point_calc_tracer {
+                    inflation_point_calc_tracer(&SkippedReason::ZeroCreditsAndReturnCurrent.into());
+                }
                 return (0, self.credits_observed);
             } else {
+                if let Some(inflation_point_calc_tracer) = inflation_point_calc_tracer {
```

Comment:
We shouldn't ever be hitting this case, right?

Reply:
Right, we shouldn't ever be hitting. :) That's one of the things I actually want to reaffirm, if there should be any oddities in the distant future. :)

Reply:
(Also, I think we're nearing to the phase where we can remove this dead altogether safely)